src/dstats.jl

```diff
@@ -1,4 +1,4 @@
-mutable struct DDEStats
+mutable struct DStats
```

Comment:
Can you explain the reasoning for renaming the struct? It's internal anyway, so the name should not matter?

Reply:
And the name `DDEStats` seems quite fitting.

Reply:
Ah maybe you just did a quick search/replace `destats` -> `stats`? In that case, can you revert these undesired changes?